plugins/woocommerce/src/Internal/RestApi/Routes/V4/Settings/PaymentGateways/Schema/ChequeGatewaySettingsSchema.php

```
@@ -0,0 +1,74 @@
+class ChequeGatewaySettingsSchema extends AbstractPaymentGatewaySettingsSchema {
+
+	/**
+	 * Get custom groups for the cheque gateway.
+	 *
+	 * Provides design-aligned labels and descriptions for the check payment
+	 * settings form fields.
+	 *
+	 * @param WC_Payment_Gateway $gateway Gateway instance.
+	 * @return array Custom group structure.
+	 */
+	protected function get_custom_groups_for_gateway( WC_Payment_Gateway $gateway ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$gateway'. (undefined)

(UnusedFormalParameter)